third_party/txt/src/txt/typeface_font_asset_provider.h

```diff
@@ -39,7 +39,7 @@ class TypefaceFontStyleSet : public SkFontStyleSet {
   int count() override;
 
   // |SkFontStyleSet|
-  void getStyle(int index, SkFontStyle*, SkString* style) override;
+  void getStyle(int index, SkFontStyle* style, SkString* name) override;
```

Comment:
Can you document what this method does with the `name` parameter. It isn't clear why name is passed into a method called `getStyle` especially since the name isn't used to determine the style produced.

This parameter is not in `SkFontStyleSet`.

Reply:
This is an implementation of an method defined by `SkFontStyleSet`:
    https://github.com/google/skia/blob/master/include/core/SkFontMgr.h#L25

The parameter names came from:
    https://github.com/google/skia/blob/master/src/ports/SkFontMgr_custom.h#L107

AFAICT this is an optional parameter that font style set implementations can use to provide a name for a style for debugging purposes.  I don't know of any official docs from Skia about this parameter.

Reply:
Ahh ok, well in any case, may be useful to document that to reduce need to sleuth in future.